docs/notebooks/06_differential_expression.ipynb

```diff
@@ -0,0 +1,1306 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def fix_sample_ids(pb):
</pre><p>not sure why this is needed? doesn't the decoupler pseudobulk functions generate useful sample ids? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>